src/Utils/BlockTemplateUtils.php

```diff
@@ -223,8 +224,16 @@ public static function build_template_result_from_file( $template_file, $templat
 		$template->area           = 'uncategorized';
 
 		// Force the Mini-Cart template part to be in the Mini-Cart template part area.
-		if ( 'wp_template_part' === $template_type && 'mini-cart' === $template_file->slug ) {
-			$template->area = 'mini-cart';
+		// @todo When this class is refactored, move title, description, and area definition to the template classes (CheckoutHeaderTemplate, MiniCartTemplate, etc).
```

Comment:
Just wondering if we can refactor the code into a function, allowing us to reuse it for other templates.

```php

function getTemplateAreaBySlug($slug) {
    $areas = [
        'mini-cart' => 'mini-cart',
        'checkout-header' => 'header'
    ];
    return $areas[$slug] ?? null;
}

if ('wp_template_part' === $template_type) {
    $template->area = getTemplateAreaBySlug($template_file->slug);
}

```

Reply:
Yes, but origami are going to be refactoring all of this soon so best we don't touch it at this point.